ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -126,55 +126,120 @@ struct FluentButtonBody: View {
         let isDisabled = !isEnabled
         let isFloatingStyle = tokens.style.isFloatingStyle
         let shouldUsePressedShadow = isDisabled || isPressed
+        let iconColor: UIColor
+        let titleColor: UIColor
+        let borderColor: UIColor
+        let backgroundColor: UIColor
+        if isDisabled {
+            iconColor = tokens.disabledIconColor
+            titleColor = tokens.disabledTitleColor
+            borderColor = tokens.disabledBorderColor
+            backgroundColor = tokens.disabledBackgroundColor
+        } else if isPressed {
+            iconColor = tokens.highlightedIconColor
+            titleColor = tokens.highlightedTitleColor
+            borderColor = tokens.highlightedBorderColor
+            backgroundColor = tokens.highlightedBackgroundColor
+        } else {
+            iconColor = tokens.iconColor
+            titleColor = tokens.titleColor
+            borderColor = tokens.borderColor
+            backgroundColor = tokens.backgroundColor
+        }
 
-        return HStack(spacing: tokens.interspace) {
-            if let image = state.image {
-                Image(uiImage: image)
-                    .resizable()
-                    .foregroundColor(Color(isDisabled ? tokens.disabledIconColor :
-                                            (isPressed ? tokens.highlightedIconColor : tokens.iconColor)))
-                    .frame(width: tokens.iconSize, height: tokens.iconSize, alignment: .center)
-                }
-
-            if let text = state.text {
-                Text(text)
-                    .fontWeight(.medium)
-                    .multilineTextAlignment(.center)
-                    .scalableFont(font: tokens.textFont,
-                                  shouldScale: !isFloatingStyle)
-                    .modifyIf(isFloatingStyle, { view in
-                        view.frame(minHeight: tokens.textMinimumHeight)
-                    })
-                }
+        @ViewBuilder
+        var buttonContent: some View {
+            HStack(spacing: tokens.interspace) {
+                        if let image = state.image {
```

Comment:
A lot of the spacing is off in this file. Could you do a `^I` on the whole file to align it to Apple standard?